storage.py

```diff
@@ -385,7 +395,8 @@ def create_user_entry(self, user_id: str):
         """Create new user entry and init fields."""
         self._data[user_id] = {
             self.POINTS_USED : 0,
-            self.POINTS_RECEIVED : 0
+            self.POINTS_RECEIVED : 0,
+            self.PM_PREFERENCE: 1
```

Comment:
Does this work with the table schema that exists already @MattPitlyk ?  Do we have to worry about this setting defaulting back to some value each day?

Reply:
This should be fine. The daily resets are for explicitly specified fields, so if we don't put the preference setting in the daily rest function, it won't be reset.